kolibri/plugins/coach/assets/src/routes/index.js

```diff
@@ -64,6 +65,14 @@ export default [
       titleParts: ['activityLabel', 'CLASS_NAME'],
     },
   },
+
+  {
+    path: '/:classId/learners',
```

Comment:
I'd add 
`name: 'CLASSLEARNERSLIST' `
so future references to the router are easiere